chainntnfs/neutrinonotify/neutrino.go

```diff
@@ -347,20 +336,27 @@ out:
 					// the script is found in a block.
 					params := n.p2pNode.ChainParams()
 					_, addrs, _, err := txscript.ExtractPkScriptAddrs(
-						msg.pkScript, &params,
+						msg.PkScript, &params,
+					)
+					if err != nil {
+						chainntnfs.Log.Error(err)
+					}
+
+					// If the historical dispatch finished
+					// without error, we will invoke
+					// UpdateConfDetails even if none were
+					// found. This allows the notifier to
+					// begin safely updating the height hint
+					// cache at tip, since any pending
+					// rescans have now completed.
+					err = n.txConfNotifier.UpdateConfDetails(
+						*msg.TxID, confDetails,
 					)
 					if err != nil {
 						chainntnfs.Log.Error(err)
 					}
 
 					if confDetails != nil {
-						err := n.txConfNotifier.UpdateConfDetails(
```

Comment:
with this change, can the below code be removed?

Reply:
wdym?

Reply:
I was thinking that the code staring below `// If we can't fully dispatch` maybe could be removed now, but it looks like it must be there to update the scan filter with the address.